package.json

```diff
@@ -74,10 +74,12 @@
     "ember-resolver": "^5.0.1",
     "ember-sinon": "~2.2.0",
     "ember-source": "~3.3.0",
+    "ember-source-channel-url": "^1.1.0",
```

Comment:
Was being used here

https://github.com/simplabs/ember-simple-auth/blob/84ca953ec7ac2f3db41ebb378b3c4d68208a64bd/config/ember-try.js#L3

but not actually included with the project. It may have been a dependency of something at one time, but a fresh install from the lockfile in master reveals that it doesn't end up being in the project

```
yarn why ember-source-channel-url
yarn why v1.9.4
[1/4] 🤔  Why do we have the module "ember-source-channel-url"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
error We couldn't find a match!
✨  Done in 1.30s.
```

Reply:
👍